docs/cli.md

```diff
@@ -93,6 +94,29 @@ In the case where your application resides in a directory other than your reposi
 folder, you can specify that directory using the `--context-dir` parameter. The
 specified directory will be used as your application root folder.
 
+#### Injecting directories to build
+
+In case you want to inject files that should be available only during the
+build time (iow. when the `assemble` script is invoked), you can specify the
+directories from where the files will be copied into the container that runs
+this script. To do so you can invoke S2I as following:
+
+```console
+$ s2i build --inject /mydir:/container/dir file://source builder-image output-image
+```
+
+In this case the content of the `/mydir` directory will get copied into
+`/container/dir` inside the container which runs the assemble script.
+After the `assemble` script finish, all files copied will be truncated and thus
+not available in the output image. The files are truncated instead of deleted
+because the user under which we run the container with assemble script might not
+have permissions to delete files in the destination directory (eg. `/etc/ssl`).
+
+You can also specify multiple directories, for example: `--inject /dir1:/container/dir1 --inject /dir2:container/dir2`.
```

Comment:
1. Can the directory in the container be relative, or must it be an absolute path?
2. The example uses `/container/dir1` and `container/dir2` (note missing slash).